src/pages/EventPage/EventPage.tsx

```diff
@@ -44,6 +45,18 @@ const EventPage: FC = () => {
     return "";
   }, [location.state]);
 
+  const [initialSession, initialSeconds] = useMemo(() => {
+    const { s, t } = queryString.parse(location.search);
+    return [s, t].map((el) => {
+      if (el !== null && typeof el === "string") {
+        const n = parseFloat(el);
+        return isNaN(n) ? 0 : n;
+      } else {
+        return 0;
+      }
```

Comment:
My only question for the whole pull request is "what happens if someone shares `url?s=1` will that route to session 1 and default to 0 seconds? AND vice-versa, if someone only shares the seconds `url?t=100` will that default to session=0?

I think this is the code block that does that but just want to make sure.

Reply:
>shares url?s=1 will that route to session 1 and default to 0 seconds?

Yes, but s=1 will route to session 2 (s starts at 0).

>shares the seconds url?t=100 will that default to session=0?

Yes

This only happens if the user alters the shareable URL though. The generated shareable URL will contain `s` and `t` (if start at checkbox is checked and t>0)